build/roslib.js

```diff
@@ -1191,8 +1206,17 @@ Topic.prototype.subscribe = function(callback) {
 /**
  * Unregisters as a subscriber for the topic. Unsubscribing will remove
  * all subscribe callbacks.
- */
-Topic.prototype.unsubscribe = function() {
+ *
+ * @param callback - the optional callback to unregister, if
+ *     * provided and other listeners are registered the topic won't
+ *     * unsubscribe, just stop emitting to the passed listener
+ */
+Topic.prototype.unsubscribe = function(callback) {
+  if (callback) {
+    this.off('message', callback);
```

Comment:
`Topic.prototype.off` should be [around in node](https://github.com/asyncly/EventEmitter2/blob/master/lib/eventemitter2.js#L422)